src/get-started/npm-script.md

```diff
@@ -5,4 +5,31 @@ title: Get Started > NPM Script
 
 # NPM Script
```

Comment:
FWIW, the [official docs](https://docs.npmjs.com/misc/scripts) refer to this as npm-scripts. I don't know if that's the capitalization and hyphenation we'd want necessarily, but "scripts" rather than "script" is more appropriate (the URL of the page should change, too).